frontend/package.json

```diff
@@ -24,6 +24,7 @@
     "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.0",
     "@types/react-jsonschema-form": "^1.7.4",
+    "@types/react-select": "^5.0.1",
```

Comment:
Hmm it's late so I may be missing something, where is this used?

Reply:
I think I got an indication from VS code to update versions at some point. I reverted the file to dev and everything worked, so I made changed it back to dev's version